libraries/AP_Motors/AP_MotorsCoax.cpp

```diff
@@ -122,8 +122,8 @@ void AP_MotorsCoax::output_to_motors()
             rc_write(AP_MOTORS_MOT_2, calc_pwm_output_1to1(_pitch_radio_passthrough, _servo2));
             rc_write(AP_MOTORS_MOT_3, calc_pwm_output_1to1(_roll_radio_passthrough, _servo3));
             rc_write(AP_MOTORS_MOT_4, calc_pwm_output_1to1(_pitch_radio_passthrough, _servo4));
-            rc_write(AP_MOTORS_MOT_5, _throttle_radio_min);
-            rc_write(AP_MOTORS_MOT_6, _throttle_radio_min);
+            rc_write(AP_MOTORS_MOT_5, get_pwm_min());
+            rc_write(AP_MOTORS_MOT_6, get_pwm_min());
```

Comment:
since the functions above are `calc_pwm_output_*`, shouldn't this be `get_pwm_output_min`?